packages/grafana-ui/src/components/Badge/Badge.story.tsx

```diff
@@ -15,7 +16,7 @@ export default {
     color: { control: 'select' },
     text: { control: 'text' },
   },
-};
+} as Meta;
```

Comment:
Is this `as Meta` needed? Does the story build fail otherwise?

Reply:
Ah well spotted @joshhunt! I originally needed it because I was using a decorator defined in the story config. It shouldn't be needed now I've switched it out for the `withCenteredStory` decorator.